builtin/logical/pki/acme_billing_test.go

```diff
@@ -147,12 +147,12 @@ func validateClientCount(t *testing.T, client *api.Client, mount string, expecte
 	// Validate this mount's namespace is included in the namespaces list,
 	// if this is enterprise. Otherwise, if its OSS or we don't have a
 	// namespace, we default to the value root.
```

Comment:
this comment is now out of date.

Reply:
It is... kinda... The empty namespace path is empty, but it is still the root namespace. :D 

But yes, I could update in a future PR when I'm making other changes, would be good to un-break enterprise now though.